modules/aws-load-balancer-operator-install.adoc

```diff
@@ -0,0 +1,286 @@
+// Module included in the following assemblies:
+//
+// * networking/aws-load-balancer-operator.adoc
+
+:_content-type: PROCEDURE
+[id="aws-load-balancer-operator-install_{context}"]
+= Installing an AWS Load Balancer Operator
+Users that wish to install an AWS Load Balancer Operator (ALBO) can do so as long as the following prerequisites are met:
```

Comment:
```suggestion
You can install an AWS Load Balancer Operator (ALBO) if you meet the following prerequsites.
```
- Per ISG, don't use `wish`. Use `want`. But I would consider rewording the sentence to make it more "you"-centric. 
- Remove the colon before a headig.

Reply:
Done.